content/html/concepts/elements/terms/input/input.md

```diff
@@ -52,15 +52,15 @@ The following examples uses a `<form>` element that features `<input>` elements:
   <body>
     <form>
       <!-- This input type requires the user to enter a valid email address -->
-      <label>Email Address</label>
+      <label for ="email" id="email">Email Address</label>
```

Comment:
Shouldn't the `id` attribute be on the `<input>` tag rather than the `<label>` tag?

Reply:
True